subprojects/language-java/src/main/java/org/gradle/api/tasks/compile/CompileOptions.java

```diff
@@ -60,11 +59,6 @@
 
     private ForkOptions forkOptions = new ForkOptions();
 
-    private boolean useDepend;
-
-    @SuppressWarnings("deprecation")
-    private DependOptions dependOptions = new DependOptions();
-
```

Comment:
OK, was wondering what purpose it really serves now that we removed all the deprecated functionality. Maybe it should become deprecated now though no really pressure to really do it.